tendermint/tests/support/lite/many_header_bisection/happy_path.json

```
@@ -1,5 +1,5 @@
 {
- 	"description": "Case: Trusted height=1, bisecting to verify height=12, should not expect error",
+ 	"description": "Case: Trusted height=1, bisecting to verify height=11, should not expect error",
```

This should actually be 10 now:
```suggestion
 	"description": "Case: Trusted height=1, bisecting to verify height=10, should not expect error",
```